api/HTMLElement.json

```diff
@@ -462,6 +459,81 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "dialog_elements": {
+          "__compat": {
+            "description": "<code>beforetoggle</code> event fires at dialog elements",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLElement/beforetoggle_event",
+            "spec_url": "https://html.spec.whatwg.org/multipage/indices.html#event-beforetoggle",
+            "support": {
+              "chrome": {
+                "version_added": false
```

Comment:
Same here:
```suggestion
                "version_added": "132"
```

Reply:
I'm OK with this. BCD do testing against the release anyway, so this would get updated if it is wrong when the release comes out. Thanks.